web/app/models.py

```diff
@@ -174,6 +174,8 @@ def to_dict(self):
 KOMI_VALUES = ['0', '0.5', '1', '1.5', '2', '2.5', '3.0', '3.5', '4', '4.5',
     '5', '5.5', '6', '6.5', '7', '7.5', '8', '8.5', '9', '9.5']
 TIE_BREAKS = ['SOS', 'SODOS', 'Face to Face Result', 'Random Procedure']
+MATCH_RESULTS = ['White Wins', 'Black Wins', 'Jigo', 'No Result',
```

Comment:
Yeah, from the guide:
"The program must provide options for several possible game outcomes: White wins, Black wins, Jigo, No result, White forfeit, Black forfeit, or Double forfeit." 
I haven't thought about scores. I'll have to dive into the guide some more. 
The MATCH_RESULTS can obviously be swapped out to reflect accepted abbreviations